cogs/ahk/ahk.py

```diff
@@ -263,6 +257,22 @@ async def ahk(self, ctx, *, code):
 
 		await ctx.send(out)
```

Comment:
I suggest:
1. have `cloudahk_call` just return the result dict
2. make a new helper method that takes that result and crafts a string that you can ctx.send
3. have the separate commands call that helper function and send using ctx.send locally

Reply:
Why? I get that passing `ctx` is a little gross, but considering that 99% of the code is shared between the commands, I don't see much of a point in doing it any other way.